sdk/storage/Azure.Storage.Queues/src/QueueClient.cs

```diff
@@ -1707,7 +1921,7 @@ protected internal virtual QueueClient WithClientSideEncryptionOptionsCore(Clien
             TimeSpan? timeToLive = default,
             CancellationToken cancellationToken = default) =>
             await SendMessageInternal(
-                ToBinaryData(messageText),
+                ToBinaryData(messageText ?? string.Empty),
```

Comment:
`ToBinaryData` used to be null tolerant.

Reply:
It was giving me a bunch of null pointers when I was trying to fix the tests.

Reply:
weird. one thing I'd check is if we can fallback to null/empty somewhere downstream from here. So that we avoid allocating empty BinaryData.

Reply:
I wonder if rest api accepts `<QueueMessage></QueueMessage>`  (i.e. without `<MessageText>` node). Could you check what happens on master if you attempt to send `null` as `messageText` ?

Reply:
TODO run this experiment.